requirements.txt

```diff
@@ -8,5 +8,6 @@ matplotlib
 numpy
 ophyd
 pymongo
```

Comment:
We need to clean up the requirements to only leave the needed ones. If something is not needed to _run_ the package, but is needed for testing, please move to the `requirements-dev.txt` file.

I think from the list, we only need to leave `ophyd`, `numpy` (? - need to check where it's used), and `pypylon`. We don't need the `tfs-pandas` here as the package is not used in any way here.

Reply:
Oh, in https://github.com/BNL-ATF/ophyd-basler/blob/master/ophyd_basler/basler_handler.py `h5py` and `area-detector-handlers` are used, so let's add them too.

Also, in https://github.com/BNL-ATF/ophyd-basler/blob/master/ophyd_basler/basler_camera.py, we use `event-model`, so let's add it as well.